### DISCUSSION TITLE
Maps in plot?

### DISCUSSION THREAD

**Discussion by mberg** [observablehq/plot#1094] [General]
Hi all - huge fan of plot and the direction it's going!

I am curious if support for maps is something that is on the roadmap for plot?   [Bertin](https://observablehq.com/@neocartocnrs/hello-bertin-js) looks extremely promising especially since it has an API that allows you to build maps in a very similar way to mapbox.  Having mapping integrated into a common chart library especially for things like facets, etc would be very compelling.  If only @mbostock and @Fil had any experience in this area :)

**Comment by Fil** [comment]
Our current effort is the Plot.carto plugin: https://observablehq.com/@fil/plot-carto-0-5 ; Bertin is certainly an inspiration and I think there will be more collaboration/mashups between the projects.

**Comment by mberg** [comment]
Thank you this looks great! Where is the repo for carto?

**Comment by Fil** [comment]
No repo yet, it's developed as a notebook.
Message ID: ***@***.***
.com>

**Comment by Fil** [comment]
This is now fully addressed with the Plot.geo mark and the top-level projection option.

**Reply by dgwyer** [reply]
This is great. Are there any examples of this implemented in the wild yet that you're aware of? 🙂

**Reply by Fil** [reply]
There are quite a few [on observable](https://observablehq.com/search?query=%22Plot.geo%22). I particularly enjoy [this one](https://observablehq.com/@fgregg/1983-mayoral-election-dot-density-maps), but if you want wild, check out the [mystical rodent prophecies](https://observablehq.com/@mjbo/determining-the-accuracy-of-mystical-rodent-prophecies) notebook!

**Reply by dgwyer** [reply]
Fantastic!

**Comment by mberg** [comment]
We will have some shortly which we can share.  We are really excited to try
this out. Thank you for all the great work on this!

**Reply by dgwyer** [reply]
Would love to see those, if you can share. Thanks.
